onnx_caffe2/backend.py

```diff
@@ -629,10 +624,14 @@ def _all_names_in_graph(graph):
         return names
 
     @classmethod
-    def onnx_graph_to_caffe2_net(cls, graph_def, device="CPU"):
+    def onnx_model_to_caffe2_net(cls, model, include_inputs_and_initializers, device="CPU"):
```

Comment:
make it the last argument (as some callsites might pass device already) and default to true

Reply:
the function name has been changed, this will be a breaking change anyway (but to be fair this function is never meant to be a public function)

Reply:
it'd actually invoked (we should use '_' prefix for private stuff): https://fburl.com/8z9yn2ck and https://github.com/ezyang/onnx-pytorch/search?utf8=%E2%9C%93&q=onnx_graph_to_caffe2_net&type=